pkg/proxy/iptables/proxier.go

```diff
@@ -651,6 +620,68 @@ func (proxier *Proxier) OnEndpointsUpdate(allEndpoints []api.Endpoints) {
 	proxier.deleteEndpointConnections(staleConnections)
 }
 
+// Gather information about all the endpoint state for a given api.Endpoints.
+func accumulateEndpointsMap(endpoints *api.Endpoints, hostname string,
+	curEndpoints map[proxy.ServicePortName][]*endpointsInfo,
+	newEndpoints *map[proxy.ServicePortName][]*endpointsInfo,
+	svcPortToInfoMap *map[proxy.ServicePortName][]hostPortInfo,
+	staleConnections *map[endpointServicePair]bool,
+	activeEndpoints *map[proxy.ServicePortName]bool) {
```

Comment:
I know it's obnoxious in the callee, but it makes the call-site so much more obvious, and since maps are absolutely unique in this regard, prefer to to be explicit?